server/engine/src/main/java/org/eclipse/lsp/cobol/cli/command/CliAnalysis.java

```diff
@@ -72,6 +72,12 @@ public class CliAnalysis implements Callable<Integer> {
     @CommandLine.ArgGroup(exclusive = false)
     private ExtendedSourceConfig extendedSourceConfig = new ExtendedSourceConfig();
 
+    @CommandLine.Option(
```

Comment:
Will it make more sense to name it warm JIT or something? Or is there other use case for it?